docs/020-deployment/ama/020-ama-second-step.md

```diff
@@ -203,29 +203,27 @@ For the list of firewall rules that should be added to your Azure Firewall, see
 
 ## Configure network settings
 
-CluedIn is very flexible in terms of network configuration. If you have any network-related questions, contact one of our infrastructure experts.
+CluedIn is very flexible in terms of network configuration and comes out of the box with a new vnet with a predefined address space and subnet. CluedIn is isolated out of the box, and this should work for most setups, but we do allow granular control over the network to support zero-trust and integrations.
+
+If you prefer to use an existing vnet, or would like to do some advanced configuration please contact one of our infrastructure experts before installation to ensure it's properly implemented as it can be challenging to change post-deployment.
 
 ### Define VNet
 
-During the installation process, CluedIn will install a newly created Azure VNet using an address space 10.0.0.0/8 and a subnet 10.0.0.0/16 for the Azure Kubernetes Service.
+During the installation process, you will have the option to update the vnet from the default address space of 10.0.0.0/8 and a subnet 10.0.0.0/16 for the Azure Kubernetes Service. 
 
 If you have an existing VNet and you want to reuse it for CluedIn installation, contact one of our infrastructure experts for assistance. You can specify the existing VNet in the <a href="/deployment/azure-marketplace/step-3#review-the-advanced-configuration-tab">Advanced configuration</a> step of CluedIn installation.
 
 ### Analyze CluedIn network configuration
 
-CluedIn comes with a default network configuration. Changing network settings after installation is a very tedious task. Therefore, make sure that you have analyzed how you want to set up the CluedIn instance at the network level before starting the installation process.
-
 The following diagram shows default CluedIn network configuration after installation.
 
 ![ama-network-1.jpeg](../../assets/images/ama/install-guide/ama-network-1.jpeg)
 
-As part of the AKS managed cluster, AMA or AKS (public cluster) is deployed as a single standard Public Azure Load Balancer for both egress and ingress traffic.
+The deployed Azure Kubernetes Service is deployed with a single Azure load balancer that is used for both ingress and egress traffic. It is possible to update this post-deployment so that access is only possible via internal vnet integration, but does require a number of steps.
```

Comment:
```suggestion
The deployed Azure Kubernetes Service is deployed with a single Azure Load Balancer that is used for both ingress and egress traffic. It is possible to update this post-deployment so that access is only possible via internal vnet integration, but it does require a number of steps.
```